templates/adv_im_calc.sl.template

```diff
@@ -45,12 +43,14 @@ if [[ $success == 0 ]]; then
     if [[ ! -d {{sim_dir}}/ch_log ]]; then
         mkdir {{sim_dir}}/ch_log
     fi
-
-    fd_name=`python -c "from qcore import utils; params = utils.load_sim_params('{{sim_dir}}/sim_params.yaml'); print(params.FD_STATLIST)"`
+    fd_name=$(getFromYaml "{{sim_dir}}/sim_params.yaml" FD_STATLIST)
     fd_count=`cat $fd_name | wc -l`
 
     # log metadata
-    python $gmsim/workflow/metadata/log_metadata.py {{sim_dir}} $step_name cores={{np}} fd_count=$fd_count models="{{models}}"start_time=$start_time end_time=$end_time status="COMPLETED"
+    python $gmsim/workflow/metadata/log_metadata.py {{sim_dir}} $step_name cores={{np}} fd_count=$fd_count models="{{models}}" start_time=$start_time end_time=$end_time status="COMPLETED"
+    # tar the temporary outputs from analysis
+    # following IM_calc_dir/station/model/component_dir
+    find {{sim_IM_calc_dir}}/*/*/ -mindepth 1 -maxdepth 1 -type d -exec bash -c 'tar --remove-files -uvf {{sim_IM_calc_dir}}/adv_im_out.tar "{}" ' \;
```

Comment:
I believe we didn't just want to tar them as it creates a lot of output not just n_files?